tracer/src/Datadog.Trace/Ci/Coverage/Exceptions/PdbNotFoundException.cs

```diff
@@ -2,25 +2,25 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
 
 using System;
 using System.Runtime.CompilerServices;
 
-namespace Datadog.Trace.Ci.Coverage.Exceptions
+namespace Datadog.Trace.Ci.Coverage.Exceptions;
```

Comment:
You might want to add `[DoesNotReturn]` on the `Throw` method, it'll help elsewhere

Reply:
done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)